### ISSUE TITLE
what user_query can i use?

### ISSUE THREAD

**Issue by xiaoyazhu** [FoundationVision/Groma#23]
If i want to locate a specific target ,such as "a person wearing a yellow hat", what user_query can i use in inference?
```
python -m groma.eval.run_groma \
    --model-name {path_to_groma_7b_finetune} \
    --image-file {path_to_img} \
    --query {user_query} \
    --quant_type 'none' # support ['none', 'fp16', '8bit', '4bit'] for inference
```

**Comment by machuofan** [comment]
You can simply use `Locate <p> a person wearing a yellow hat </p> in the image.` or something else like that. Just remember to enclose the referring expression with `<p>` and `</p>`.

**Comment by xiaoyazhu** [comment]
But when i tested several images, i found that whether there is an object in the test image, the model will output a localization result. Is there any way to adjust the settings , or is it caused by model hallucination ?

**Comment by machuofan** [comment]
Yes, such hallucination is probably caused by training data - for grounding training, we only got positive QA pairs, i.e., the object mentioned in the question is guaranteed to occur in the image. To remedy such hallucination, you can curate some negative QA pairs and finetune the model.

**Comment by Eman-Abdelrahman** [comment]
If I need to fine-tune the model for locating target elements in an image, is there certain hyper parameters to focus on, please?

**Comment by LLH-Harward** [comment]
If I want to provide the object's label and bounding box to Groma to help generate more accurate image descriptions, how should I structure the prompt?
